### PR TITLE
IZE-531 unit test for schema package

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- add a unit test for the Validate function

### CODE REVIEW

File: internal/commands/ci.go [hunk 1]
```diff
@@ -105,6 +96,11 @@ func (o *CIOptions) Run(cmd *cobra.Command) error {
 		return err
 	}
 
+	key, err := getPublicKey(fmt.Sprintf("%s/.ssh/id_rsa.pub", o.Config.Home))
```
Comment (AutomationD) [inline internal/commands/ci.go:99]: Let's add a TODO to read the key as a parameter too.

File: internal/commands/tfenv.go [hunk 2]
```diff
@@ -118,7 +113,7 @@ func GenerateTerraformFiles(project *config.Project, terraformStateBucketName st
 				return err
 			}
 
-			// If we haven't found an existing legacy format state bucket use a <NAMESPACE>-<AWS_ACCOUNT>-tf-state bucket as default (unless overriden with other parameters.
+			// If we haven't found an existing legacy format state bucket use a <NAMESPACE>-<AWS_ACCOUNT>-tf-state bucket as default (unless override with other parameters).
```
Comment (AutomationD) [inline internal/commands/tfenv.go:116]: Incorrect correction. See https://www.wordnik.com/words/overridden

**Review by AutomationD — CHANGES_REQUESTED** [review]
I would not call this PR just unit test, but refactor+unit test.

**Comment by AutomationD** [comment]
Let's address minor questions, Otherwise looks good 👍🏻

File: internal/commands/ci.go (hunk 1 above)
Comment (psihachina) [reply internal/commands/ci.go:99]: I created a story in Jira

**Review by AutomationD — APPROVED** [review]

**Pull request merged**